wordcab_transcribe/utils.py

```diff
@@ -60,7 +61,66 @@
     "₿",
 ]
 
+# run API for youtube video endpoint
+def run_api_youtube(url : str, sourceLang = "en", timestamps = "s", wordTimestamps = False):
+    headers = {"accept": "application/json", "Content-Type": "application/json"}
+    params = {"url": url}
+    data = {
+        "alignment": True,  # Longer processing time but better timestamps
+        "diarization": True,  # Longer processing time but speaker segment attribution
+        "source_lang": sourceLang,  # optional, default is "en"
+        "timestamps": timestamps,  # optional, default is "s". Can be "s", "ms" or "hms".
+        "word_timestamps": wordTimestamps,  # optional, default is False
+    }
+
+    response = requests.post(
+        "http://localhost:5001/api/v1/youtube",
+        headers=headers,
+        params=params,
+        data=json.dumps(data),
+    )
+
+    r_json = response.json()
+
```

Comment:
A response status check should ensure you get a 200 response. If 200, you can return the JSON response. Otherwise, you must warn the user.